fe/src/main/cup/sql_parser.cup

```diff
@@ -1515,6 +1523,81 @@ partition_key_desc ::=
     :}
     ;
 
+fixed_partition_key_desc ::=
+    LBRACKET partition_values:keys RPAREN
+    {:
+        if (keys.size() != 2) {
+            throw new Exception("partition range must contains two elements.");
+        }
+        if (keys.get(0).toUpperCase().equals("MAXVALUE")) {
+            throw new Exception("the left value can't MAXVALUE.");
+        }
```

Comment:
You should not check semantic here, this should be done in analyze phase in FE.
Here should only check syntax.